hello-spring/pom.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+>
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <!-- lookup parent from repository -->
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.6.2</version>
+    <relativePath />
+  </parent>
+
+  <groupId>org.folio</groupId>
+  <artifactId>folio-hello-spring</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>hello-spring</name>
+  <description>Hello world demo project for FOLIO, using Spring and OpenAPI</description>
+
+  <licenses>
+    <license>
+      <name>Apache License 2.0</name>
+      <url>https://spdx.org/licenses/Apache-2.0</url>
+    </license>
+  </licenses>
+
+  <properties>
+    <java.version>11</java.version>
+    <folio-spring-base.version>3.0.0</folio-spring-base.version>
+    <openapi-generator.version>5.3.0</openapi-generator.version>
+  </properties>
+
+  <dependencies>
+    <!-- Imports most of the required dependencies -->
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-spring-base</artifactId>
+      <version>${folio-spring-base.version}</version>
+    </dependency>
+
+    <!-- Currently only required because Spring looks for a database on startup, despite no actual use -->
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.openapitools</groupId>
+      <artifactId>jackson-databind-nullable</artifactId>
+      <version>0.2.1</version>
```

Comment:
Can you bump the version from 0.2.1 to 0.2.2 to fix dependency vulnerabilities?
https://mvnrepository.com/artifact/org.openapitools/jackson-databind-nullable/0.2.1
https://mvnrepository.com/artifact/org.openapitools/jackson-databind-nullable/0.2.2